testnet/docker-compose.local.yml

```diff
@@ -19,11 +19,11 @@ services:
       dockerfile: ./dockerfiles/enclave.Dockerfile
       args:
         TESTMODE: true
-  enclave-debug:
-    image: "testnetobscuronet.azurecr.io/obscuronet/enclave_debug:latest"
-    build:
-      context: $ROOT_PATH
-      dockerfile: ./dockerfiles/enclave.debug.Dockerfile
+#  enclave-debug:
```

Comment:
Maybe we should delete if this isn't used/is causing issues?

Reply:
not causing issues. But it increases the build times